service/http-api-samples/http-api-keystore-sample/pom.xml

```diff
@@ -0,0 +1,104 @@
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <artifactId>http-api-samples</artifactId>
+    <groupId>org.fido.iot</groupId>
+    <version>0.3-SNAPSHOT</version>
+  </parent>
+
+  <name>HTTP Keystore API</name>
+  <artifactId>http-api-keystore-sample</artifactId>
+  <packaging>jar</packaging>
+
+  <properties>
+    <jakson.dataformat.cbor.version>2.11.0</jakson.dataformat.cbor.version>
+    <commons-codec.version>1.14</commons-codec.version>
+    <tomcat.version>9.0.37</tomcat.version>
+    <bcpkix.version>1.65</bcpkix.version>
+    <dbcp2.version>2.7.0</dbcp2.version>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.fido.iot</groupId>
+      <artifactId>cert-utils</artifactId>
+      <version>0.3-SNAPSHOT</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.commons</groupId>
+      <artifactId>commons-dbcp2</artifactId>
+      <version>${dbcp2.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>com.h2database</groupId>
+      <artifactId>h2</artifactId>
+      <version>1.4.200</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.junit.jupiter</groupId>
+      <artifactId>junit-jupiter</artifactId>
+      <scope>test</scope>
+    </dependency>
+
+    <dependency>
+      <groupId>org.bouncycastle</groupId>
+      <artifactId>bcpkix-jdk15on</artifactId>
+      <version>${bcpkix.version}</version>
+    </dependency>
+
+    <dependency>
+      <groupId>org.apache.tomcat.embed</groupId>
+      <artifactId>tomcat-embed-core</artifactId>
+      <version>${tomcat.version}</version>
+    </dependency>
+  </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.1.2</version>
+        <executions>
+          <execution>
+            <id>copy-dependencies</id>
+            <phase>package</phase>
+            <goals>
+              <goal>copy-dependencies</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/lib/</outputDirectory>
+              <includeScope>runtime</includeScope>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>exec-maven-plugin</artifactId>
+        <version>3.0.0</version>
+        <executions>
+          <execution>
+            <goals>
+             <goal>exec</goal>
+            </goals>
+          </execution>
+        </executions>
+        <configuration>
+          <executable>java</executable>
+          <includeProjectDependencies>true</includeProjectDependencies>
+          <workingDirectory>${project.build.outputDirectory}</workingDirectory>
+          <mainClass>org.fido.iot.sample.DiClientApp</mainClass>
```

Comment:
The exec main class is set as DiClientApp for every API class. This needs to be updated for API samples.